rpcs3/main.cpp

```diff
@@ -95,7 +97,9 @@ LOG_CHANNEL(q_debug, "QDEBUG");
 		if (AttachConsole(ATTACH_PARENT_PROCESS) || AllocConsole())
 			[[maybe_unused]] const auto con_out = freopen("conout$", "w", stderr);
 #endif
-		std::fprintf(stderr, "RPCS3: %.*s\n", static_cast<int>(text.size()), text.data());
+		std::unique_lock lock(stderr_mutex);
+		std::cerr << "RPCS3: " << text << '\n';
```

Comment:
In my experience std::cerr is usually "atomic" if you use it with a single object, like,
std::cerr << text << std::endl;
Idk why there's a need for a mutex